src/Features/LanguageServer/Protocol/CustomProtocol/FindUsagesLSPContext.cs

```diff
@@ -210,15 +211,17 @@ public override async ValueTask OnReferenceFoundAsync(SourceReferenceItem refere
             {
                 if (documentSpan != default)
                 {
-                    // We do have a source span, so compute location normally.
+                    // We do have a document span, so compute location normally.
                     return await ProtocolConversions.DocumentSpanToLocationAsync(documentSpan, cancellationToken).ConfigureAwait(false);
                 }
 
-                // If we have no source span, our location may be in metadata.
+                // If we have no document span, our location may be in metadata or may be a namespace.
                 var symbol = await SymbolFinder.FindSymbolAtPositionAsync(document, position, cancellationToken).ConfigureAwait(false);
-                if (symbol == null || symbol.Locations == null || symbol.Locations.IsEmpty || !symbol.Locations.First().IsInMetadata)
+                if (symbol == null || symbol.Locations.IsEmpty || symbol.Kind == SymbolKind.Namespace)
```

Comment:
i'm a bit confused by this.  so what happens if someone does Find-Usages on a namespace?